core/package.json

```diff
@@ -99,7 +100,7 @@
   "bugs": {
     "url": "https://github.com/ionic-team/ionic/issues"
   },
-  "homepage": "https://github.com/ionic-team/ionic#readme",
+  "homepage": "https://ionicframework.com/",
```

Comment:
https://ionicframework.com/docs

?

Reply:
For the core? For the ionic/docs I have it at `https://ionicframework.com/docs`